flipt-engine/gen/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "protos"
```

Comment:
this doesnt get built into the `engine` lib correct? just want to ensure the engine libraries are as small as they can be since they are embedded in all the clients

Reply:
With this goal we don't need `tonic`. I built release for main and this branches. It's results.
```
// main
768B libfliptengine.d
8.2M libfliptengine.dylib
1.5M libfliptengine.rlib

// this 
928B libfliptengine.d
8.3M libfliptengine.dylib
1.3M libfliptengine.rlib
```